src/main/java/com/sap/sgs/phosphor/fosstars/tool/github/SecurityRatingCalculator.java

```diff
@@ -27,33 +40,59 @@
 
   /**
    * Entry point.
+   *
+   * @param args Command-line parameters.
+   */
+  public static void main(String... args) {
+    try {
+      run(args);
+    } catch (Exception e) {
+      System.out.printf("[x] Something went wrong!%n");
+      e.printStackTrace(System.out);
```

Comment:
e.printstacktrace() by default prints it to System.out isnt it?

Reply:
Nope, it uses `System.err` by default. In fact, I am planning to update it to use loggers, see #35.